packages/manager/apps/hpc-vmware-managed-vcd/src/components/vrackNetwork/VrackNetworkDatagridSubDatagrid.component.tsx

```diff
@@ -66,30 +66,42 @@ export default function VrackNetworkDatagridSubDatagrid({
           {
             id: 'actions',
             label: '',
-            cell: (network: string) => (
-              <div className="flex items-center justify-end">
-                <OdsButton
-                  aria-hidden="true"
-                  label=""
-                  aria-label={t(
-                    'managed_vcd_dashboard_vrack_network_delete_network',
+            cell: (network: string) => {
+              const isDeleting = vrackSegmentStatus === 'deleting';
+              const buttonId = `delete-network-${network}`;
+              return (
+                <div className="flex items-center justify-end">
+                  <OdsButton
+                    id={buttonId}
+                    aria-hidden="true"
+                    label=""
+                    aria-label={t(
+                      'managed_vcd_dashboard_vrack_network_delete_network',
+                    )}
+                    icon="trash"
+                    variant="ghost"
+                    isDisabled={isDeleting}
+                    onClick={() => {
+                      const networkIp = network.split('/')[0];
```

Comment:
May be hash you use it as identifier for url. Create function "createIdentifier" or just use `encodeURIComponent`

https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent